bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetEnergyHandler.java

```diff
@@ -70,6 +76,9 @@ public OpenWebNetEnergyHandler(Thing thing) {
     public void initialize() {
         super.initialize();
 
+        Object refreshPeriodConfig = getConfig().get(OpenWebNetBindingConstants.CONFIG_PROPERTY_REFRESH_PERIOD);
+        ENERGY_REFRESH_PERIOD = Integer.parseInt(refreshPeriodConfig.toString());
```

Comment:
Don't use a static (constant) if you want to set it.
Your constant should rather be the default value to be used in case the parameter is not set properly.

Reply:
Good point I changed ENERGY_REFRESH_PERIOD to  a non  static variable energyRefreshPeriod that  is set by xml parameter